sdcm/utils/common.py

```diff
@@ -1671,7 +1671,8 @@ def get_branched_ami(scylla_version: str, region_name: str, arch: AwsArchType =
     else:
         images = [ec2_resource.images.filter(Filters=filters), ]
     images = sorted(itertools.chain.from_iterable(images), key=lambda x: x.creation_date, reverse=True)
-    images = [image for image in images if not image.name.startswith('debug-')]
+    images = [image for image in images if not image.name.startswith(
+        'debug-') or image.name.contains(f'{ScyllaProduct}-debug-)')]
```

Comment:
@yaronkaikov this code works ? I don't see `ScyllaProduct` being defined anywhere, where do you expect it to come from ?